### PR TITLE
[feature](mtmv)(1)remove old mtmv code

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Issue Number: close #xxx

remove old mtmv code,we will implement mtmv in a new way

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Review by dataroaring — CHANGES_REQUESTED** [review]

**Review by morningman — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/persist/OperationType.java [hunk 1]
```diff
@@ -271,18 +271,6 @@ public class OperationType {
...
-
-    public static final short OP_CREATE_MTMV_TASK = 340;
-    public static final short OP_DROP_MTMV_TASK = 341;
-    @Deprecated
-    public static final short OP_CHANGE_MTMV_TASK = 342;
-
-    public static final short OP_ALTER_MTMV_STMT = 345;
-
     public static final short OP_DROP_EXTERNAL_TABLE = 350;
     public static final short OP_DROP_EXTERNAL_DB = 351;
```
[the remark this answers is not in the document]
Comment (morningman) [reply fe/fe-core/src/main/java/org/apache/doris/persist/OperationType.java:275]: Agreed, we can mark this as deprecated and just read and discard these logs

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by zddr** [comment]
run p0

**Comment by morningman** [comment]
run buildall

**Comment by zddr** [comment]
run feut

**Comment by zddr** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by morningman — APPROVED** [review]

**Pull request merged**